scripts/cloudfront_routing.js

```
@@ -0,0 +1,54 @@
+/**
+ * This script follows ES5.1 rules for compatibility.
+ * - Avoids ES6+ syntax (e.g., `let`, `const`, arrow functions).
+ */
+'use strict';
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)